src/display/svg.js

```diff
@@ -464,14 +462,9 @@ var SVGGraphics = (function SVGGraphicsClosure() {
 
       return this.loadDependencies(operatorList).then(function () {
         this.transformMatrix = IDENTITY_MATRIX;
-        this.pgrp = document.createElementNS(NS, 'svg:g'); // Parent group
-        this.pgrp.setAttributeNS(null, 'transform', pm(viewport.transform));
-        this.tgrp = document.createElementNS(NS, 'svg:g'); // Transform group
-        this.tgrp.setAttributeNS(null, 'transform', pm(this.transformMatrix));
         this.defs = document.createElementNS(NS, 'svg:defs');
-        this.pgrp.appendChild(this.defs);
-        this.pgrp.appendChild(this.tgrp);
-        this.svg.appendChild(this.pgrp);
+        this.svg.setAttributeNS(null, 'transform', pm(viewport.transform));
```

Comment:
this does not work for chrome -- we will need to create a root group and use it as this.svg ?